README.md

```diff
@@ -57,6 +56,19 @@ To run the integration tests, you must run it on the command line with the follo
 
 ```dotnet test Amazon.QLDB.Driver.IntegrationTests --settings Amazon.QLDB.Driver.IntegrationTests/.runsettings```
 
+## Getting Help
+
+Please use these community resources for getting help.
+* Ask a question on StackOverflow and tag it with the [amazon-qldb](https://stackoverflow.com/questions/tagged/amazon-qldb) tag.
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
+* If you think you may have found a bug, please open an [issue](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues/new).
+
+## Opening Issues
+
+If you encounter a bug with the Amazon QLDB Dotnet Driver, we would like to hear about it. Please search the [existing issues](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues) and see if others are also experiencing the issue before opening a new issue. When opening a new issue, we will need the version of Amazon QLDB Dotnet Driver, Dotnet language version, and OS you’re using. Please also include reproduction case for the issue when appropriate.
+
+The GitHub issues are intended for bug reports and feature requests. For help and questions with using AWS SDK for GO please make use of the resources listed in the [Getting Help](https://github.com/awslabs/amazon-qldb-driver-dotnet#getting-help) section. Keeping the list of open issues lean will help us respond in a timely manner.
+
 ### Documentation 
```

Comment:
documentation is a subsection fo development

Reply:
Moved